### PR TITLE
bugfix/13473-scrollbar-threshold

### LINKED ISSUE #13473: disappearing column/bar when scrolling beyond point value
#### Expected behaviour
All points should be visible after scrolling.

#### Actual behaviour
When you scroll out the chart so the first point will be outside of the scroll area it will disappear after scrolling back.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/x40da2gq/
Steps: 
1/ Scroll fully to the right, 
2/ Return to the starting position
3/ The first point won't be visible anymore

#### Workaround
After setting the min value everything is working correctly:
https://jsfiddle.net/BlackLabel/tz69qwa1/

#### Product version
8.1.0

#### Affected browser(s)
All

### PR DESCRIPTION
**Pull request by khlieng**
Fixed #13473, it was not possible to scroll back down to the `threshold` after scrolling up with `yAxis.scrollbar` enabled and no explicit `yAxis.min` set.

### CODE REVIEW

File: ts/Core/Axis/ScrollbarAxis.ts [hunk 1]
```diff
@@ -87,13 +87,15 @@ class ScrollbarAxis {
                             Math.min(
                                 axisMin,
                                 axis.min as any,
-                                axis.dataMin as any
+                                (axis.dataMin as any) > (axis.threshold as any) ?
+                                    axis.threshold : (axis.dataMin as any)
```
Comment (TorsteinHonsi) [inline ts/Core/Axis/ScrollbarAxis.ts:91]: Wouldn't it be more compact and readable to write `Math.min(axis.threshold, axis.dataMin)` in this expression and the three others?

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]

File: ts/Core/Axis/ScrollbarAxis.ts (hunk 1 above)
Comment (khlieng) [reply ts/Core/Axis/ScrollbarAxis.ts:91]: `Math.min` return `NaN` if any of the values are undefined.

Comment (khlieng) [reply ts/Core/Axis/ScrollbarAxis.ts:91]: Maybe something like `Math.min(pick(axis.threshold, Number.MAX_VALUE), axis.dataMin)` could work though.

Comment (khlieng) [reply ts/Core/Axis/ScrollbarAxis.ts:91]: Or not, tests are failing because `Math.min` turns `null` into `0` :/

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**